proposed/event-dispatcher-meta.md

```diff
@@ -7,7 +7,7 @@ The purpose of this document is to describe the rationale and logic behind the E
 
 ## 2. Why Bother?
 
-Many libraries, components, and frameworks have long supported mechanisms for allowing arbitrary third party code to interact with them.  Most are variations on the classic Observer pattern, often mediated through an intermediary object or service.  Others take a more Aspect-Oriented Programming (AOP) approach.  Nonetheless all have the same basic concept: interupt program flow at a fixed point to provide information to arbitrary third party libraries with information about the action being performed and allow them to either react or influence the program behavior.
+Many libraries, components, and frameworks have long supported mechanisms for allowing arbitrary third party code to interact with them.  Most are variations on the classic Observer pattern, often mediated through an intermediary object or service.  Others take a more Aspect-Oriented Programming (AOP) approach.  Nonetheless all have the same basic concept: interrupt program flow at a fixed point to provide information to arbitrary third party libraries with information about the action being performed and allow them to either react or influence the program behavior.
```

Comment:
What changed in this paragraph? I've read through it twice, and I cannot for the life of me figure out what it might have been.

Suggestion: can we wrap lines at 80 characters, please? Doing so makes identifying changes in paragraphs simpler, as only the lines with changes will be marked in the diff, not the entire paragraph.

Reply:
Um.  Probably whitespace is my guess?  Not sure.  And hard-wrapping the text makes editing and updating it a huge PITA, because every edit requires reflowing every line.  That's why I try to avoid it.